app/common/AreaFilterSelect.jsx

```diff
@@ -42,18 +42,40 @@ class AreaFilterSelect extends React.Component {
       return null;
     };
 
-    return (
-      <div className="area-filter-select">
-        <Select
-          name="area-filter-select"
-          placeholder="Filter by Administrative Boundary"
-          value={this.state.selectedLayer}
-          options={this.props.options}
-          onChange={this.handleSelectChange}
-        />
-        { polygonSelectorComponent() }
-      </div>
-    );
+    const {
+      exploreANeighborhoodSelected,
+      ntaSelection,
```

Comment:
Looks like `ntaSelection` is never used?

Reply:
correct, it was part of a failed experiment. Consider it gone!